docs/wiki/ALZ-Contribution-Guide.md

```diff
@@ -0,0 +1,183 @@
+
+`loadTextContent('../resources/Microsoft.Authorization/policyDefinitions/Name-Of-The-Policy.json')`
+
+For a policy set definition, additional code should be added inside of the `loadPolicySetDefinitions` variable under the correct environment:
+
+`loadTextContent('../resources/Microsoft.Authorization/policySetDefinitions/Deploy-Sql-Security.json')`
+
+The policy definition files will be compiled into a `policies.json` file from the `policy.bicep` file which was amended.
+
+Once the policy work has been completed, a pull request has been submitted to the repository:
+
+![pr-example](./media/pr-example.png)
+
+Policy versioning follows the same protocol as built-in policies. More information on that can be found in the [ALZ Policies document in the wiki](./ALZ-Policies.md#versioning).
+
+For policy deprecation, the process is documented in the [Azure Landing Zones - Deprecating Policies](./ALZ-Deprecated-Services.md) page.
+
+If a policy is part of an initiative, references to policies that are being deprecated should be removed. Policy initiatives are located in the [policySetDefinitions](../../src/resources/Microsoft.Authorization/policySetDefinitions/) folder. To find out if a policy is part of an initiative it is recommended to look up the policy definition in [AzAdvertiser](http://azadvertizer.com/) and check for association with initiatives. When identified, go into the necessary initiative and remove references to the definition. Locate the policy definition in the parameters of the initiative and remove reference:
+
+![Example policy def in initiative](./media/example-def-in-init.png)
+
+Also find it in the policyDefinitions and remove reference as well:
+
+![Example policy def in initiative 2](./media/example-def-in-init-2.png)
+
+When working within the policy files, to read parameters which are set at the top level of the policy definition a double escape is needed for ARM. So instead of using `[parameters('someParameter')]` within the policy, you should use `[[parameters('someParameter')]` instead.
+
+> **Note:** When testing the policy manually in the portal or another deployment outside of the ALZ Accelerator (Portal), you will need to remove the double escaping, `[[`, and revert to normal ,`[`'
+
+When working with policies that are assigned by default, these are located under the [eslzArm/managementGroupTemplates/policyAssignments](../../eslzArm/managementGroupTemplates/policyAssignments) folder. References to policy definitions are done through the assignments, so if any amendments are done to default assigned policies, they should be amended here too. A wiki to default assignments can be found [in the wiki](./ALZ-Policies.md).
+
+Policies in `eslzArm.json` file will also need updating if wanting to assign a new policy that is located. The file for this amendment [is located here](../../eslzArm/eslzArm.json).
```

Comment:
Instead of using `here` in the link, can we use the path `eslzArm/eslzArm.json` so someone reading can see where "here" actually is?

Reply:
Amended